test/helper.js

```diff
@@ -0,0 +1,17 @@
+const mvm = require('mongodb-version-manager');
+const Promise = require('bluebird');
+
+before(function() {
+  this.timeout(1000000);
+
+  // Set default MONGODB_VERSION to be checked against later
+  process.env.MONGODB_VERSION = process.env.MONGODB_VERSION || 'stable';
+
+  console.log('Installing MongoDB server');
+
+  return Promise.promisify(mvm.use)({
```

Comment:
This feels like a stretch just to use promises, can't we just provide a `done` in `before`s function:
```
before(function(done) {
   // everything else

   mvm.use({ version: ... }, done);
});
```

Reply:
Yeah, I understand I can revert the change. I had it on my todo list to eventually play around with mongodb-version-manager and promisify it's interface. I'll revert it and make my changes here once completed.